### ISSUE TITLE
value-error missing from aria-describedBy

### ISSUE THREAD

**Issue by jackheslop96** [alphagov/accessible-autocomplete#752]
In our most recent DAC audit, a AAA was reported relating to:

`aria-describedBy=value__assisstiveHint` on the accessible autocomplete input.

It was suggested we add:

`aria-describedBy=value__assisstiveHint value-error`
